doc/rst/developer/bestPractices/git.rst

```diff
@@ -444,7 +446,7 @@ Other logging commands
 To view commits grouped by author (for example, show me commits by author from
 1.9.0.1 tag to now):
```

Comment:
The second to last paragraph above this section needs two ` around ``git push origin main``